app/redux/actions.js

```diff
@@ -44,3 +44,11 @@ export function stopCpuProfiling() {
     }
   };
 }
+
+export function updateCPUProfilingTable(cpuProfilingTable) {
+  return {
+    type: 'UPDATE_CPU_PROFILING_TABLE',
```

Comment:
I may be wrong here, but this looks like a catch-all action. What exactly gets updated here? Does this add more data to the profiling table? or does it modify the profiling table data structure in some way?